### PR TITLE
archive old demos

### PR DESCRIPTION
**Pull request by tharvik**
update drynx, stainless, and spindle demo page to point to the archived git.

### CODE REVIEW

File: views/products/demo/drynx.tpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p>Together with our SwissRe partner we created a
-    <a href="https://factory.c4dt.org/incubator/drynx/demo">Demonstrator</a> using the Drynx data analysis tool.
+    Demonstrator using the Drynx data analysis tool.
```
Comment (ineiti) [inline views/products/demo/drynx.tpl:2]:
```suggestion
    demonstrator using the Drynx data analysis tool.
```

File: views/products/demo/drynx.tpl [hunk 2]
```diff
@@ -27,9 +27,9 @@
...
-intuitively correct. To test it out yourself, you can visit the
-<a href="https://factory.c4dt.org/incubator/drynx/demo">Drynx Demonstrator</a></p>
+intuitively correct.</p>
 
-<a href="https://factory.c4dt.org/incubator/drynx/demo" class="tab-link">
-    For a Live Demo Click Here
-</a>
+<p>
+    We do not host this demo anymore but you can
+    <a href="https://github.com/c4dt/service-drynx">find its code here</a>.
```
Comment (ineiti) [inline views/products/demo/drynx.tpl:34]: The `service-drink` needs to have a README with the following information:
- link to the showcase
- link to the paper
- explanation how to run the demo
- 1 or 2 screenshots

**Review by ineiti — CHANGES_REQUESTED** [review]
For both demoes we should have a built docker-image somewhare (github or docker.io) so that people don't need to build the image themselves.

**Comment by tharvik** [comment]
> For both demoes we should have a built docker-image somewhare (github or docker.io) so that people don't need to build the image themselves.

that's already the case (but not documented I agree); as theses two demos use a plurality of docker, I was thinking of adding a docker-compose (and not only a list of docker images), would that work for you?

**Comment by ineiti** [comment]
Of course, docker-compose.yaml is a very good idea. But if possible we should host the images somewhere, so that people don't need to do a `docker-compose build` first. Or, well, wait for the build if they do `up`.

**Comment by ineiti** [comment]
Waiting for https://github.com/c4dt/TODO/issues/327

File: views/products/demo/drynx.tpl (hunk 2 above)
Comment (PascalinDe) [reply views/products/demo/drynx.tpl:34]: I followed the README and have a local installation (which throws some errors, but I'm not familiar enough with the code to know if they are "normal"), and there's a link to the showcase as well as a screenshot, but no link to the paper.

Comment (ineiti) [reply views/products/demo/drynx.tpl:34]: Correct - added it to the https://github.com/c4dt/TODO/issues/345

File: views/products/demo/stainless-for-smart-contracts.tpl [hunk 3]
```diff
@@ -26,6 +26,8 @@
     the user to have a background in software development and blockchain
     technology to fully benefit from it.
 </p>
-<a href="https://factory.c4dt.org/incubator/stainless/demo/" class="tab-link">
-    For a Live Demo Click Here
-</a>
+
+<p>
+    We do not host this demo anymore but you can
+    <a href="https://github.com/c4dt/service-stainless">find its code here</a>.
```
[the remark this answers is not in the document]
Comment (ineiti) [reply views/products/demo/stainless-for-smart-contracts.tpl:32]: Absolutely, see https://github.com/c4dt/service-drynx/issues/48

Sorry this falls on you - I would've preferred we finish this with the old team :)

For tomorrow's C4DT Factory Update I would like to have the minimal text up, but we'll need to finish the issues.

**Review by ineiti — APPROVED** [review]

File: views/products/demo/stainless-for-smart-contracts.tpl (hunk 3 above)
[the remark this answers is not in the document]
Comment (PascalinDe) [reply views/products/demo/stainless-for-smart-contracts.tpl:32]: I was talking about service-stainless - I opened a PR with some details on the project and installation instructions as well

the docker-compose is not working for that project either (I update https://github.com/c4dt/service-stainless/issues/114 with my problems which are different from yours)

**Pull request merged**